pom.xml

```diff
@@ -0,0 +1,98 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>it.pagopa.interop.probing.eservice.registry.updater</groupId>
+	<artifactId>interop-probing-eservice-registry-updater</artifactId>
+	<version>0.0.1-SNAPSHOT</version>
+	<name>interop-probing-eservice-registry-updater</name>
+	<description>Register Interop E-Service Probing</description>
+	<organization>
+		<name>PagoPa</name>
+		<url>https://www.pagopa.it</url>
+	</organization>
+	<properties>
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+		<maven.compiler.source>17</maven.compiler.source>
+		<maven.compiler.target>17</maven.compiler.target>
+		<java.version>17</java.version>
+		<mockito.version>4.8.1</mockito.version>
+		<slf4j.version>2.0.6</slf4j.version>
+	</properties>
+	<dependencies>
+		<dependency>
+			<groupId>io.awspring.cloud</groupId>
+			<artifactId>spring-cloud-starter-aws-messaging</artifactId>
+			<version>2.3.0</version>
```

Comment:
Use properties for version's packages better if parent pom

Reply:
same of the reader